http3/roundtrip.go

```diff
@@ -143,6 +149,7 @@ func (r *RoundTripper) RoundTripOpt(req *http.Request, opt RoundTripOpt) (*http.
 	if err != nil {
 		return nil, err
 	}
+	defer cl.useCount.Add(-1)
```

Comment:
We reduce the count here, out of the lock. Unless we add a lock here, or we should use atomic.